### DISCUSSION TITLE
PowerShell

### DISCUSSION THREAD

**Discussion by SoniEx2** [MonoGame/MonoGame#8441] [General]
Is it possible to use PowerShell with MonoGame?

**Comment by stromkos** [comment]
What do you mean "Use PowerShell"?

Yes it is possible to call `dotnet` commands from powershell.

PowerShell has its own environment and parameter handling and certain expectations of CMD may not hold.

**Reply by SoniEx2** [reply]
As in write a game in PowerShell+MonoGame, without C#

**Comment by stromkos** [comment]
No.  I don't think so.  

Back in the day, I did some fairly unorthodox COM calls from within MS Access to synchronize users from an AS400, NT4, and Novell Netware "domains". 

Therefore, I am unwilling to to give a definitive no. 

That being said the language can be any .Net6+ compatible languages, like VB, F# or MS IL could be used instead of C#, assuming feature parity.

**Reply by SoniEx2** [reply]
is PowerShell Core not a .Net6+ compatible language?

**Reply by stromkos** [reply]
I do not think it can be compiled into IL.

**Reply by SoniEx2** [reply]
it might need a wrapper shim to get the PowerShell going, yes. what's the issue?
